### ISSUE TITLE
App writes too many information on logs (logcat)

### ISSUE THREAD

**Issue by Simonmicro** [Freeyourgadget/Gadgetbridge#1058]
#### Your issue is:
I have disabled the gadgetbridge logs inside the settings - there are still many notifications inside the logs, when connected to a device. See the screenshot. There should be no entries specally no entries with the debugging flag! This would improve the battery lifetime with enabled bluetooth and connected gadgetbridge - because the system/logd service would not wake up the device so often anymore.
![screenshot_20180406-164139](https://user-images.githubusercontent.com/24966116/38427823-8cbcf498-39ba-11e8-9de6-0390521ad8d1.png)

#### Your wearable device is:
HPlus - But I think it is not relevant.

#### Your android version is:
7.1.2

#### Your Gadgetbridge version is:
0.25.1

**Comment by IzzySoft** [comment]
Are you sure you don't confuse Gadgetbridge logs (which are written to GB's own directory) with system logs (logcat)? The screenshot looks like the latter, so you might wish to see e.g. [Setting Android Log Levels](https://stackoverflow.com/q/10289129/2533433).

**Comment by Simonmicro** [comment]
Well, I've recorded the system logs. But that will expand if I activate the Gadgetbridge debug mode (= logs active). So I guess Gadgetbridge will still send some notifications to the logsystem, still if it is not in debug mode - which should not happen. And yes, I've enabled the verbose log mode, so I can list all information, generated by Gadgetbridge. My device is rooted and I'm using the CatLog app.
Best regards,
Simonmicro
